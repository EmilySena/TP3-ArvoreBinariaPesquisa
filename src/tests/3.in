i alghunxaslkdhae
i alghunxaslkdhai
i alghunxaslkdhaa
i abghunxaslkdhaa
i azghunxaslkdhaa
i a
i ac
i z
i ax
e 4 a ac ax z
s z z
e 4 a ac ax z
s alghunxaslkdhae alghunxaslkdhae
e 4 a ac ax z
d 2 9 8
s ax zz
d 2 9 8
