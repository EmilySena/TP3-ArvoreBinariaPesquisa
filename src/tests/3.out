4 5 8 9
4 5 8 9
3 4 8 9
z ax
zz z
